### DISCUSSION TITLE
Searching for individual tracks by tags in the database

### DISCUSSION THREAD

**Discussion by remus-selea** [jcorporation/myMPD#1349] [Q&A] [answered]
How can searching be done for individual tracks within all albums in the database?  I couldn't find a way to search for individual tracks. For instance, some albums might have only 1-2 acoustic folk songs, with the rest another genre like metal. If I want to listen to just the acoustic folk tracks across my entire music database, how can I exclude the others?

If this were possible it should also enable searching for individual tracks based on their tags, even within large albums, such as if you had a single massive `[non-album tracks]` album and only want a few certain tracks from it.

It would enable me to do more advanced searches for individual tracks using genre tags. For instance, I want to filter for tracks that are tagged as both "traditional" and "Japanese," and thus exclude those that are tagged as "traditional" but that also include "French," "Italian," or other tags instead of "Japanese."

--- 
I found that in settings you can add `Title` for `Tags to browse`
![image](https://github.com/user-attachments/assets/446a030c-e36c-4730-8b88-a65d66f9c6f0)

This adds searching by title.
![image](https://github.com/user-attachments/assets/443e6fdf-1250-414a-bb99-6e39590c434c)
However in this view you don't have the option to search using the comparison operators like there is in the browsing by `Album`.

![image](https://github.com/user-attachments/assets/5124fc70-b25a-4991-932b-0bd248b4a9b7)

**Comment by jcorporation** [comment, marked as answer]
You should use the song search feature for this.

**Reply by remus-selea** [reply]
That’s a bit embarrassing! I find the user interface a bit confusing. I've only searched in the `Browse` tab and managed to completely miss the other section. Thanks for pointing it out!

Here it is in case anyone else is wondering.
![image](https://github.com/user-attachments/assets/adac7c82-9392-4428-8e15-9fb46b198da3)

--- 
As a side note, I see an opportunity for new contributors to improve the documentation for existing myMPD features, such as the search functionality. 

Thank you for all your efforts on myMPD—it's definitely my favorite music player to self-host.

**Reply by jcorporation** [reply]
I appreciate all contributions.
